### PR TITLE
Fix SFC without a script tag show an error

### LINKED ISSUE #1187: SFC without a script tag show an error when trying to import
- [x] I have searched through existing issues
- [x] I have read through [docs](https://vuejs.github.io/vetur)
- [x] I have read [FAQ](https://github.com/vuejs/vetur/blob/master/docs/FAQ.md)

## Info

- Platform: macOS
- Vetur version:0.18.1
- VS Code version: 1.33.0

## Problem

Components without a script tag and an export are flagged with an error:

>File '...import_example/src/components/Simple.vue' is not a module. Vetur(2306)

```html
<!-- src/components/Simple.vue -->
<template>
  <h1>No script tag in this component</h1>
</template>
```

```javascript
// App.vue
// ...
import Simple from "./components/Simple.vue";
// ...
```

## Reproducible Case

Here is the diff to a brand new project created with vue-cli:

```diff
diff --git a/src/App.vue b/src/App.vue
index 7633616..04a51d9 100644
--- a/src/App.vue
+++ b/src/App.vue
@@ -1,17 +1,17 @@
 <template>
   <div id="app">
     <img alt="Vue logo" src="./assets/logo.png">
-    <HelloWorld msg="Welcome to Your Vue.js + TypeScript App"/>
+    <Simple/>
   </div>
 </template>
 
 <script lang="ts">
 import { Component, Vue } from 'vue-property-decorator';
-import HelloWorld from './components/HelloWorld.vue';
+import Simple from "./components/Simple.vue";
 
 @Component({
   components: {
-    HelloWorld,
+    Simple,
   },
 })
 export default class App extends Vue {}
diff --git a/src/components/Simple.vue b/src/components/Simple.vue
new file mode 100644
index 0000000..a7ad133
--- /dev/null
+++ b/src/components/Simple.vue
@@ -0,0 +1,3 @@
+<template>
+  <h1>No script tag in this component</h1>
+</template>
```

Here is the repo that shows the problem:

https://github.com/uri/vetur-import-example

### PR DESCRIPTION
**Pull request by yoyo930021**
Fixed #1187

### CODE REVIEW

**Comment by NickBolles** [comment]
LGTM!

File: server/src/embeddedSupport/embeddedSupport.ts [hunk 1]
```diff
@@ -138,6 +138,10 @@ export function getSingleTypeDocument(
     }
   }
 
+  if (type === 'script' && newContent.trim().length <= 0) {
```
Comment (NickBolles) [inline server/src/embeddedSupport/embeddedSupport.ts]: <= 0 seems a little defensive. length can never be less than 0, so just `=== 0` should be fine

Comment (yoyo930021) [reply server/src/embeddedSupport/embeddedSupport.ts]: It's my small habit. XD

**Comment by heivo** [comment]
When will this be merged, is there anything missing?

**Comment by webcoderkz** [comment]
It's june 2020 already, still not merged, do they plan to merge it? It's annoying.

**Comment by yoyo930021** [comment]
Hi anyone, I publish self-maintenance vscode extension.
This extension will be maintained until Vetur is reactivated.

https://marketplace.visualstudio.com/items?itemName=yoyo930021.vuter

**Comment by octref** [comment]
Thanks a lot and sorry for the slow response.

**Pull request merged**